src/router/userRoute.js

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express'
```

Comment:
Rename `userRoute.js` to `user.js` or if you prefer to `userRoutes.js` (plural), and after move it to `router/routes` directory.